CheckoutTests/Tokenisation/CheckoutAPIServiceTests.swift

```diff
@@ -200,3 +205,106 @@ final class CheckoutAPIServiceTests: XCTestCase {
     XCTAssertEqual(subject.correlationID, stubCorrelationID)
   }
 }
+
+extension CheckoutAPIServiceTests {
```

Comment:
Can we add a unit test, when `securitycode` is empty or invalid length (for instance, 6 numbers)? In this case, we will get an internal SDK error I assume.

Reply:
I had a look at this and I think for the api service itself, it's a bit of over-responsibility to run a pre-check over the provided security code. So, it's being done in `SecurityCodeComponent`. Added some invalid input tests for 5 digits into `SecurityCodeComponentTests` to cover those scenarios @chintan-soni-cko